controlplane/migrations/0055_mighty_cannonball.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS "graph_compositions" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"federated_graph_schema_version_id" uuid NOT NULL,
+	"subgraph_schema_version_id" uuid NOT NULL
+);
+--> statement-breakpoint
+DO $$ BEGIN
```

Comment:
I think now it is the right moment to move
- `router_config`
- `is_composable`

 away from `schema_versions` to `graph_compositions` so we don't have mixed responsibilities. That would require to maintain a `graph_compositions` that stores one entry per composition and `graph_composition_subgraphs` the related subgraph.

WDYT?

Reply:
Done